src/utils.coffee

```diff
@@ -180,6 +180,7 @@ process_layer = (layer)->
 
 exports.build_upload_params = (options) ->
   params =
+    quality_override: utils.as_safe_bool(options.quality_override)
```

Comment:
The type of the parameter is string (like quality).

The list of parameters is sorted. This should be in line 200

Reply:
Sure, we'll change it.